content/en/tracing/guide/trace_ingestion_volume_control.md

```diff
@@ -24,6 +24,8 @@ If you decide to reduce the ingestion volume for certain services, the **request
 
 **Note**: If your applications and services are instrumented with OpenTelemetry libraries and you set up sampling at the SDK level and/or at the collector level, APM metrics are based on the **sampled** set of data by default. Use the [Datadog Processor][17] to keep the metrics accuracy, while using collector-level sampling. See [Ingestion Sampling with OpenTelemetry][4] for more information.
 
+<div class="alert alert-info"><strong>Beta</strong>: Alternatively, use the <a href="https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/datadogconnector">Datadog Connector</a> to calculate APM metrics on unsampled data. Read <a href="/opentelemetry/guide/switch_from_processor_to_connector">Switch from Datadog Processor to Datadog Connector for OpenTelemetry APM Metrics</a> for more information.</div>
```

Comment:
@gord02 - Should this say "sampled data" instead of "unsampled data"?

Reply:
It should say un-sampled data